Automattic/wp-calypso
# Reader post card - move title and excerpt above images

Related to # pe7F0s-1fb-p2 

## Proposed Changes

* Updates the reader post cards to show the title and excerpt above the image or gallery.

Before:

<img width="487" alt="Screenshot 2023-09-13 at 2 27 20 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/a2704f3a-36f6-4a0d-87d8-a109d4ff49f6">


After:

<img width="489" alt="Screenshot 2023-09-13 at 2 27 53 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/32aef7d5-9627-4b21-8fff-75ae1cb82cb5">


## Testing Instructions



* Test this in the reader.
* View streams with non-compact post cards (recent, search, etc.).
* verify the title and except are above the image
* verify the various paddings/spacings make sense above and below the image.
* Smoke test other streams in the reader, compact cards, etc. to verify we havent accidently broken any styles with the titles and excerpts.

## Pre-merge Checklist
